knowledge_repo/app/migrations/versions/eb39ac93fc39_add_pageview_indexes.py

```diff
@@ -0,0 +1,26 @@
+"""add_pageview_indexes
+
+Revision ID: eb39ac93fc39
+Revises: d15f6cac07e1
+Create Date: 2019-11-15 15:30:55.010491
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'eb39ac93fc39'
+down_revision = 'd15f6cac07e1'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.create_index(
```

Comment:
Since it passes CI, it's probably fine, but in superset we always use `batch_alter_table`/`batch_op` since plain `op` doesn't work with some dialects.

https://alembic.sqlalchemy.org/en/latest/batch.html <- more info

Reply:
nvm